.circleci/config.yml

```diff
@@ -90,12 +90,14 @@ jobs:
       - generate_gradle_wrapper
       - run:
           name: Build the project
-          command: bash gradlew build
-      - run:
-          name: Publish the project to local maven
-          command: bash gradlew publishToMavenLocal
```

Comment:
I assume the aggregate-build-artifacts takes the place of publishing to local maven?

Reply:
This `publishToMavenLocal` step is actually not needed. It was carried over into this project from the SDK -- there, it is important.